core/src/main/java/io/kestra/core/validations/validator/FlowValidator.java

```diff
@@ -87,6 +89,33 @@ public boolean isValid(
             violations.add("Duplicate output with name [" + String.join(", ", duplicateIds) + "]");
         }
 
+        // No missing defaults for schedule triggers
+        if (!ListUtils.emptyOnNull(value.getTriggers()).isEmpty()
+            && !ListUtils.emptyOnNull(value.getInputs()).isEmpty()) {
+            // Find inputs without defaults
+            Set<String> inputsWithoutDefaults = value.getInputs().stream()
+                    .filter(input -> input.getDefaults() == null)
+                    .map(Data::getId)
+                    .collect(Collectors.toSet());   
+            // Find schedules with missing inputs or null inputs
+            value.getTriggers().stream()
+                    .filter(Schedule.class::isInstance)
+                    .map(Schedule.class::cast)
+                    .filter(schedule -> {
+                        Map<String, Object> scheduleInputs = schedule.getInputs();
+                        return scheduleInputs == null || inputsWithoutDefaults.stream()
```

Comment:
what happen when there is triggers and input, but no inputsWithoutDefaults ? better confirm that with a unit test